content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
+| [PhysX Collider](\docs\user-guide\components\reference\physx\collider) | Edit the dimensions of a collider. |
+| [PhysX Ball Joint](\docs\user-guide\components\reference\physx\ball-joint) | Edit the position and limitations of a joint. |
+
+## Manipulators 
+
+With manipulators, you can edit an entity's Transform component properties directly in the 3D Viewport instead of in the [Entity Inspector](.\..\entity-inspector).  When you select an entity, only the current transform mode's manipulator displays.  The orientation of the manipulator matches the reference space selection.
+
+### Translation manipulator
+
+![Image of the translation manipulator with the linear, planar, and surface manipulator highlighted](/images/user-guide/editor/transform-manipulator.png)
+
+The translation manipulator is composed of three different types of manipulators. The red, green, and blue arrows are _linear manipulators_, they correspond to the X-axis, Y-axis, and Z-axis, respectively.  You can **left-click** and drag a linear manipulator to move an entity along a single axis at a time.  Manipulators turn yellow when you select them.
+
+Use _Planar manipulators_, represented by squares, to move an entity along two axes while restricting movement on the unselected axis. 
+
+The yellow sphere at the center of the translation manipulator is the _surface manipulator_.  You can move an entity along the surface of other meshes that are visible in the 3D Viewport.  If there are no surfaces that the entity would intersect with, the entity will move a fixed distance in front of the camera.
+
+![Image of translation manipulator with a ghost axis](/images/user-guide/editor/ghost-axis.png)
+
+The translation manipulators display a red, blue, and green _ghost axis_ that indicates the original position of the entity while a manipulator is in use.
+
+### Rotation manipulator
+
+![Image of the rotation manipulator](/images/user-guide/editor/rotation-manipulator.png)
+
+The rotation manipulator is composed of two types of manipulators. A set of red, green, and blue circles are _angular manipulators_ that rotate the entity on the X-axis, Y-axis, and Z-axis, respectively.  The larger white circle is the _camera space manipulator_, which rotates the entity on the Editor camera's forward axis. **Left-click** and drag a manipulator to rotate the entity on an axis.
+
+![Image of the rotation manipulator with a yellow sector that shows how much the entity has been rotated](/images/user-guide/editor/rotation-feedback.png)
+
+The rotation manipulators display a yellow sector that indicates the original orientation of the entity while a manipulator is in use.
+
+### Scale manipulator
+
+![Image of the scale manipulator](/images/user-guide/editor/scale-manipulator.png)
+
+The scale manipulator is represented by a set of white cubes on an 3D axis.  You can **left-click** and drag away from the manipulator's center to increase the uniform-scale of an entity.
```

Comment:
nit:  Shouldn't this be 'a' instead of 'an'

Reply:
I think it should in fact be `an`, but I'd deffo not trust my take and instead wait for the wordsmiths to correct me 😅

Reply:
I think it should in fact be `an`, but I'd deffo not trust my take and instead wait for the wordsmiths to correct me 😅

Reply:
Fixed.